openmdao/visualization/n2_viewer/style/legend.css

```diff
@@ -1,17 +1,16 @@
 /******************* Legend styling *******************/
 #legend-div {
-	width: 90%;
-	position: absolute;
-	bottom: 20px;
+    position: absolute;
+    left: 200px;
+	top: 90%;
```

Comment:
not a big deal but maybe replaces tabs with spaces in this file, or vice versa... it's currently mixed.

Reply:
Got it, not sure how that happened...